.travis.yml

```diff
@@ -67,38 +43,36 @@ jobs:
     - CC_VER="5"
     - PLATFORM='xenial'
     script: "./travis/build.sh"
-
-  ######################## XENIAL-EMSCRIPTEN
   - stage: build+test
     os: linux
     env:
     - PLATFORM='xenial-emscripten'
     script: "./travis/build.sh"
-
-#  ######################## OSX + TESTS
-#  - stage: build+test
-#    os: osx
-#    if: branch = master
-#    env:
-#    - CMAKE_ARGS='-DBUILD_TESTS=ON'
-#    - TEST='TRUE'
-#    script:
-#    - "./travis/build.sh"
-#
-#  ######################## OSX - PYQRLLIB
-#  - stage: build+test
-#    os: osx
-#    if: branch = master
-#    env:
-#    - CMAKE_ARGS='-DBUILD_PYTHON=ON'
-#    script:
-#    - "./travis/build.sh"
-
-#  ########################
-#  - stage: test_deployment
-#    script: echo "TODO..."
-
-  ########################
+  - stage: build_deb_dist
+    before_install:
+    - openssl aes-256-cbc -K $encrypted_450b689219c8_key -iv $encrypted_450b689219c8_iv -in keys.tar.enc -out travis/keys.tar -d
+    os: linux
+    env:
+      - BUILD_DIST='TRUE'
+      - CC_VER=5
+      - PLATFORM='xenial'
+    script:
+    - "./travis/build.sh"
+    - "./travis/xenial/push_debs_github.sh"
+  - stage: build_deb_dist
+    before_install:
+    - openssl aes-256-cbc -K $encrypted_450b689219c8_key -iv $encrypted_450b689219c8_iv -in keys.tar.enc -out travis/keys.tar -d
+    os: linux
+    env:
+      - BUILD_DIST='TRUE'
+      - CC_VER=5
+      - PLATFORM='stretch'
+    script:
+    - "./travis/build.sh"
+    - "./travis/stretch/push_debs_github.sh"
+  - stage: merge_deb_platforms_to_ghpages
```

Comment:
This is enabled all the time, even PRs.. We dont want a PR to replace installation packages

Reply:
This could even be a security problem, if someone submits a variation of the travis config. Can they replace the debs?